### ISSUE TITLE
No standard for unknown admission/discharge location

### ISSUE THREAD

**Issue by alistairewj** [MIT-LCP/mimic-omop#38]
We have hospital admission source of "** INFO NOT AVAILABLE **" and a discharge location of "OTHER FACILITY". For admission, the only decent match I found was `32199` - Information not available - from class "UB04 Point of Origin". The UB04 ontology seems to be for claims though, not sure what the impact of that would be.

For the discharge of other facility I used `8844` - Other Place of Service - though there was the option of `32209` - Unknown Value (but present in data) from class "UB04 Pt dis status".

Seems odd there is no standard concept for a location of "unknown", am I missing something?

**Comment by parisni** [comment]
indeed, a "unknown" concept is missing quite everywhere. That's something
we might discuss with the omop community

2018-03-26 21:34 GMT+02:00 Alistair Johnson <notifications@github.com>:
